CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Splunk Enterprise SDK for Java Changelog
 
+## Version 1.7.2
+
+### New Features and APIs
+* Added a support to add custom headers in Service class. (Github PR [#176](https://github.com/splunk/splunk-sdk-java/pull/176)).
+* SSL Certificate validation (default implementation) added. (Github PR [#175](https://github.com/splunk/splunk-sdk-java/pull/175)).
+  * Boolean flag is introduced to skip/validate certificate.
```

Comment:
Revise to tell users what the flag is/how to use it:
* Boolean flag is introduced to skip/validate certificate: `HttpService.setValidateCertificates()`